src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -152,6 +153,22 @@ public string Assembly
             }
         }
 
+        public ElementTypeEnum ElementType
+        {
+            get
+            {
+                if (entries.All(entry => entry.ElementType == ElementTypeEnum.Package) &&
+                    subCategories.All(subCategory => subCategory.ElementType == ElementTypeEnum.Package))
+                    return ElementTypeEnum.Package;
+                else
+                    if (entries.All(entry => entry.ElementType == ElementTypeEnum.CustomDll) &&
+                        subCategories.All(subCategory => subCategory.ElementType == ElementTypeEnum.CustomDll))
+                        return ElementTypeEnum.CustomDll;
+                    else
+                        return ElementTypeEnum.RegularCategory;
+            }
+        }
```

Comment:
Should we use `Any` here instead of `All` and not iterate through all the sub elements?